diff --git a/node_modules/@eth-optimism/hardhat-ovm/README.md b/node_modules/@eth-optimism/hardhat-ovm/README.md
index 5b3cec5..5388760 100644
--- a/node_modules/@eth-optimism/hardhat-ovm/README.md
+++ b/node_modules/@eth-optimism/hardhat-ovm/README.md
@@ -26,7 +26,7 @@ import "@eth-optimism/hardhat-ovm"
 
 ## Configuration
 
-**By default, this plugin will use OVM compiler version 0.7.6**.
+**By default, this plugin will use OVM compiler version 0.6.12**.
 Configure this plugin by adding an `ovm` field to your Hardhat config:
 
 ```js
diff --git a/node_modules/@eth-optimism/hardhat-ovm/dist/index.js b/node_modules/@eth-optimism/hardhat-ovm/dist/index.js
index 34886e0..c58e969 100644
--- a/node_modules/@eth-optimism/hardhat-ovm/dist/index.js
+++ b/node_modules/@eth-optimism/hardhat-ovm/dist/index.js
@@ -32,7 +32,7 @@ const task_names_1 = require("hardhat/builtin-tasks/task-names");
 require("./type-extensions");
 const OPTIMISM_SOLC_VERSION_URL = 'https://api.github.com/repos/ethereum-optimism/solc-bin/git/refs/heads/gh-pages';
 const OPTIMISM_SOLC_BIN_URL = 'https://raw.githubusercontent.com/ethereum-optimism/solc-bin/gh-pages/bin';
-const DEFAULT_OVM_SOLC_VERSION = '0.7.6';
+const DEFAULT_OVM_SOLC_VERSION = '0.6.12';
 const getOvmSolcPath = async (version) => {
     if (process.env.__DANGEROUS_OVM_IGNORE_ERRORS__) {
         console.log('\n\n__DANGEROUS_OVM_IGNORE_ERRORS__ IS ENABLED!\n\n');
