diff --git a/node_modules/@eth-optimism/plugins/hardhat/compiler/index.js b/node_modules/@eth-optimism/plugins/hardhat/compiler/index.js
index af9453c..c20779e 100644
--- a/node_modules/@eth-optimism/plugins/hardhat/compiler/index.js
+++ b/node_modules/@eth-optimism/plugins/hardhat/compiler/index.js
@@ -33,7 +33,7 @@ require("./type-extensions");
 const OPTIMISM_SOLC_BIN_URL = 'https://raw.githubusercontent.com/ethereum-optimism/solc-bin/gh-pages/bin';
 // I figured this was a reasonably modern default, but not sure if this is too new. Maybe we can
 // default to 0.6.X instead?
-const DEFAULT_OVM_SOLC_VERSION = '0.7.6';
+const DEFAULT_OVM_SOLC_VERSION = '0.6.12';
 /**
  * Find or generate an OVM soljson.js compiler file and return the path of this file.
  * We pass the path to this file into hardhat.
